tests/integration/test_integration_env.py

```diff
@@ -0,0 +1,9 @@
+"""Test isolated environment setup for integration tests."""
+
+def test_isolated_env_does_not_touch_real_home(isolated_test_env):
```

Comment:
- Does this actually verify that the environment does not touch the "real home"?
- When you mean "real home," are you referring to the user's home directory? or `ZOWE_CLI_HOME`?